src/sidebar/services/annotations-exporter.ts

```diff
@@ -108,8 +99,90 @@ export class AnnotationsExporter {
       Total users: ${uniqueUsers.length}
       Users: ${uniqueUsers.join(', ')}
       Total annotations: ${annotations.length}
-      Total replies: ${annotations.filter(isReply).length}
+      Total replies: ${replies.length}
       
       ${annotationsText}`;
   }
+
+  buildCSVExportContent(
+    annotations: APIAnnotationData[],
+    {
+      groupName = '',
+      defaultAuthority = '',
+      displayNamesEnabled = false,
+      /* istanbul ignore next - test seam */
+      now = new Date(),
+    }: ExportOptions = {},
+  ): string {
+    const { uri, uniqueUsers, replies, extractUsername } = this._exportCommon(
+      annotations,
+      {
+        displayNamesEnabled,
+        defaultAuthority,
+      },
+    );
+
+    const escapeCSVValue = (value: string): string => {
+      // If the value contains a comma, newline or double quote, then wrap it in
+      // double quotes and escape any existing double quotes.
+      if (/[",\n]/.test(value)) {
+        return `"${value.replace(/"/g, '""')}"`;
+      }
+      return value;
+    };
+    const annotationToRow = (annotation: APIAnnotationData) =>
+      [
+        now.toISOString(),
+        uri,
+        groupName,
+        `${uniqueUsers.length}`,
+        `${annotations.length}`,
+        `${replies.length}`,
+        annotation.created,
+        isReply(annotation) ? 'Reply' : 'Annotation',
+        quote(annotation) ?? '',
+        extractUsername(annotation),
+        annotation.text,
+        annotation.tags.join(','),
+        pageLabel(annotation) ?? '',
+      ]
+        .map(escapeCSVValue)
+        .join(',');
+
+    const headers =
+      'Date,URL,Group,Total Users,Total Annotations,Total Replies,Annotation/Reply Date,Annotation/Reply Type,Quote,User,Body,Tags,Page';
```

Comment:
I had a bunch of feedback on the columns and labels themselves, but see the main discussion thread.